src/components/signupDirectory/SignupDirectory.tsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import { Card, Typography, Row, Col, Image } from 'antd';
+import styled from 'styled-components';
+import { DARK_GREY, ORANGE } from '../../colors';
+const { Text, Link } = Typography;
+
+const SignupDirectoryCard = styled(Card)`
+  margin: 24px;
+  width: 400px;
+  //border-color: #D9D9D9;
+  height: 424px;
+  transition: all 0.2s ease-in-out;
+  transform: translateY(3px);
+
+  :hover {
+    transform: translateY(0px);
+  }
+`;
+
+const SignupDirectoryCardBody = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+`;
+
+const RoundImage = styled(Image)`
+  img {
+    border-radius: 50%;
+    width: 160px;
+    height: 160px;
+  }
+  margin-bottom: 1em;
+`;
+const CardTitle = styled(Text)`
+  //Size from Figma, using Title will require overriding styles :(
```

Comment:
What does "using Title" mean?

Reply:
The Title component from AntD, the way it injects the CSS will require more complicated styling (basically using child selectors which is dependent on actual AntD implementation, yuck), so Text was used instead since its more generic.

Reply:
Sounds good 👍